### PR TITLE
Ii 7943 anomaly agent

### PR DESCRIPTION
**Pull request by fzhcary**

### CODE REVIEW

File: dell/anomaly_alert.py.txt [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+import logging
+import configparser
+from dateutil.tz import gettz
+from datetime import datetime, timedelta
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def log_in(host, user_name, password, user_agent):
```
Comment (crespo5609) [inline dell/anomaly_alert.py.txt:15]: remove the unnecessary code

File: dell/anomaly_alert.py.txt [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    startTime = int(start.timestamp()*1000)
+    endTime = int(end.timestamp()*1000)
+
+    proj_name, user_name = profile.split('@')
+
+    data = {"projectName": proj_name, "transferToProjectName": "dummy", "transferToCustomerName": "user",
+            "startTime": startTime, "endTime": endTime, "licenseKey": licenseKey}
+    user_agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.122 Safari/537.36"
+
+    # (cookies, headers) = log_in(host, user_name, password, user_agent)
```
Comment (crespo5609) [inline dell/anomaly_alert.py.txt:48]: same here

File: dell/anomaly_alert.py [hunk 3]
```diff
@@ -1,63 +1,56 @@
...
     resp = requests.get(url, params=data, verify=False)
-    logging.debug(str(resp))
     assert resp.status_code == 200, "failed to get anomaly data!"
-    return json.loads(resp.text)
+    result = {}
+    try:
+        result = resp.json()
+    except Exception as e:
+        logging.WARNING(e)
+    # logging.info(f"resp={resp}")
```
Comment (dearmore) [inline dell/anomaly_alert.py:24]: Should remove debug code

File: dell/anomaly_alert.py.txt [hunk 4]
```diff
@@ -0,0 +1,133 @@
+import requests
```
Comment (dearmore) [inline dell/anomaly_alert.py.txt:1]: This file should not be included as part of the commit.

**Review by dearmore — CHANGES_REQUESTED** [review]

File: dell/anomaly_alert.py.txt (hunk 1 above)
Comment (fzhcary) [reply dell/anomaly_alert.py.txt:15]: this has been removed in anomaly_alert.py. will remove the txt file.

File: dell/anomaly_alert.py.txt (hunk 2 above)
Comment (fzhcary) [reply dell/anomaly_alert.py.txt:48]: same as above

File: dell/anomaly_alert.py (hunk 3 above)
Comment (fzhcary) [reply dell/anomaly_alert.py:24]: done

File: dell/anomaly_alert.py.txt (hunk 4 above)
Comment (fzhcary) [reply dell/anomaly_alert.py.txt:1]: np

**Comment by fzhcary** [comment]
addressed all comments.

**Pull request closed without merging**